src/dependency-manager/src/manager.ts

```diff
@@ -123,20 +124,39 @@ export default abstract class DependencyManager {
           internal_host = interface_details.host;
           external_port = interface_details.port.toString();
           internal_port = interface_details.port.toString();
+          // external_protocol = interface_details.protocol;
+          // internal_protocol = interface_details.protofol;
+          external_protocol = 'https';
```

Comment:
@tjhiggins , @davidthor - do we want to assume `https` for `EXTERNAL_PROTOCOL` of ExternalNodes?

Reply:
Fine for now

Reply:
internal_protocol should also be https. then remove comments above

Reply:
Done